.gitignore

```diff
@@ -3,13 +3,12 @@
 *~
 *.bak
 
-# composer lockfile and installed packages
-/composer.lock
```

Comment:
Presumably we'll get a more consistent build by pinning dependencies to specific versions/commits via `composer.lock`